mak/copyimports.d

```diff
@@ -0,0 +1,30 @@
+import std.array, std.conv, std.file, std.getopt, std.path, std.stdio;
```

Comment:
Perhaps add a module documentation explaining what this script do and why it's necessary ?
```suggestion
/**
 * Helper script to copy source files to the `import` directory
 *
 * When building druntime, `src` are copied to the `import`, directory, excluding modules from the `rt` package.
 * On POSIX this is handled by a Makefile rule, however DigitalMars' `make` on Windows does not support defining
 * custom rules, originally leading to an unhealthy duplication between mak/COPY and mak/WINDOWS,
 * which this script removes.
 */
 module copyimports;
 
 import std.array, std.conv, std.file, std.getopt, std.path, std.stdio;
```

Reply:
Thanks, added the header with a few minor modifications.